### ISSUE TITLE
Mark as a bot based on a message

### ISSUE THREAD

**Issue by ChildishGiant** [Googe14/tf2-bot-kicker-gui#17]
There's a bot that on joining instantly takes a random username (I've not been able to see what it uses before it loads in) but spams "BAN USA FROM OLYMPIC GAMES!" in chat so if a user could be marked as a bot for a certain message that'd be great.

**Comment by Googe14** [comment]
Hmm, I am a little cautious of checking chat messages as they can be a lot more varied from legitimate users and so I am kinda worried about false positives. But I will definitely consider the possibility in the future, will probably just have to be strict about any regex to match chat messages against.

**Comment by ChildishGiant** [comment]
I think for the reasons you've said you'd want to only check for exact copies of messages, no substrings etc. So regex anchors would be needed (eg `^BAN USA FROM OLYMPIC GAMES!$`)

**Comment by Googe14** [comment]
Yes I think that would be a good idea. Another hurdle will be choosing the correct player, since chat messages aren't identified with a SteamID it would be looking at players by name, I think this would a non-issue though as if the player name is a duplicate we can select the player that joined more recently. 

I'll see about adding this if I have time to work on it, for now you will just have to manually flag these bots and get their SteamID.
